actuator-firmware/src/debug.c

```diff
@@ -76,3 +76,38 @@ void panic_handler(void)
     while (1) {
     }
 }
+
+void print_stack_info(void)
+{
+#if (CH_DBG_FILL_THREADS != TRUE) || (CH_CFG_USE_REGISTRY != TRUE) || (CH_DBG_ENABLE_STACK_CHECK != TRUE)
+#error "Requires: CH_DBG_FILL_THREADS CH_CFG_USE_REGISTRY CH_DBG_ENABLE_STACK_CHECK"
+#endif
+
+    BaseSequentialStream* chp = OUTPUT_STREAM;
+
+    const uint32_t STACK_FILL = 0x55555555;
```

Comment:
Consider using CH_DBG_STACK_FILL_VALUE rather than a "magic value". Will required you to move to checking byte-by-byte rather than word-by-word.

Reply:
Ok done